mythril/mythril/mythril_config.py

```diff
@@ -199,7 +199,6 @@ def set_api_rpc_localhost(self) -> None:
     def set_api_from_config_path(self) -> None:
         """Set the RPC mode based on a given config file."""
         config = ConfigParser(allow_no_value=False)
-        # TODO: Remove this after this issue https://github.com/python/mypy/issues/2427 is closed
         config.optionxform = str  # type:ignore
```

Comment:
Thanks for the contribution!
Can you remove this `# type: ignore`

Reply:
Yes. Done